src/components/sbb-map-container/sbb-map-container.stories.js

```diff
@@ -0,0 +1,81 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const fakeSituationCardStyles =
+  'background-color: var(--sbb-color-milk-default); height: 116px; display: flex; align-items: center; justify-content: center; border-radius: 8px; margin-bottom: 16px;';
+const Template = () => (
+  <sbb-map-container>
+    <div style="padding: 16px">
+      <sbb-form-field placeholder="Type location..." style="width: 100%">
+        <sbb-icon slot="prefix" name="magnifying-glass-small"></sbb-icon>
+        <input minLength="2" name="keyword" autoComplete="off" placeholder="Search" />
+        <sbb-icon
+          slot="suffix"
+          name="cross-medium"
+          className="custom-search-results__input-clear hidden"
+        ></sbb-icon>
+      </sbb-form-field>
+      <sbb-title level="4">Operations & Disruptions</sbb-title>
+      <div style={fakeSituationCardStyles}>
```

Comment:
I am a little bit lit, but why not just do an iteration and set the number with index+1?